src/panels/config/integrations/ha-config-flow-card.ts

```diff
@@ -71,13 +77,18 @@ export class HaConfigFlowCard extends LitElement {
                   ? "_self"
                   : "_blank"}
               >
-                <mwc-list-item hasMeta>
+                <mwc-list-item
+                  aria-label=${this.hass.localize(
+                    "ui.panel.config.integrations.config_entry.open_configuration_url"
+                  )}
+                  graphic="icon"
+                  hasMeta
+                >
                   ${this.hass.localize(
                     "ui.panel.config.integrations.config_entry.open_configuration_url"
-                  )}<ha-svg-icon
-                    slot="meta"
-                    .path=${mdiOpenInNew}
-                  ></ha-svg-icon>
+                  )}
+                  <ha-svg-icon slot="graphic" .path=${mdiCog}></ha-svg-icon>
+                  <ha-svg-icon slot="meta" .path=${mdiOpenInNew}></ha-svg-icon>
```

Comment:
Is this meant to convey that the link will open in new window?  If so, it's not purely decorative and needs a label.

Reply:
Yes, it is an indicator it opens a new window, will, so in that case it needs an aria label?

(Sorry I'm a newbie on this stuff).

Reply:
By the looks of it, `ha-svg-icon` forces `aria-hidden` to be true.
I have no controls to that from this end (meaning all other occurrences in our codebase are incorrect too).

Given that this is already an existing piece of code (this "open in new window" meta icon, was already there), I'm going to leave that out of this PR for now. I want to prevent refactoring structures not related to this PR.

Reply:
Yes, in a11y lingo it's known as notifying the user of a "change in context".  Plus the localization of course, add something like `aria-label="Opens in new tab", and the key probably belongs in `ui.common` since there's lots of these throughout the UI.

Reply:
Alright, that's fine.  I'll make a note to do a global search for that icon to fix in another PR.

FYI, the `aria-hidden` on the SVG only removes it and children, not the host ancestor.  That's there because the vast majority of icons in the UI are "purely decorative" in a11y speak, meaning they don't need a label because the info is conveyed already.  For example, the bookshelf icon is redundant with the "documentation" text.